votes/DMFF logo_vote.md

```diff
@@ -0,0 +1,12 @@
+# A vote for community Logo: DMFF
```

Comment:
Why is DMFF added here?

Reply:
Sorry, it will be added in another PR, deleted it now